<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>633</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kalkulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(245, 245, 250);
    font-family: Segoe UI;
    font-size: 14px;
}

QPushButton {
    background-color: rgb(220, 180, 255);
    color: black;
    border-radius: 10px;
    padding: 8px;
    font-weight: bold;
    border: 1px solid rgb(200, 150, 230);
}

QPushButton:hover {
    background-color: rgb(200, 160, 240);
}

QPushButton:pressed {
    background-color: rgb(180, 140, 220);
}

QLineEdit {
    background-color: white;
    border: 1px solid rgb(190, 190, 200);
    border-radius: 6px;
    padding: 4px;
}

QLabel {
    color: rgb(80, 80, 80);
    font-weight: 600;
}

QLCDNumber {
    background-color: black;
    color: rgb(0, 255, 127);
    border: 2px solid rgb(80, 80, 80);
    border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDodaj">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOdejmij">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMnoz">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDziel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonModulo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRownaSie">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przecinek">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wysop">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="memory">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdec">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditbin">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdithex">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditoct">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonkon">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Source Code Pro Medium&quot;;
</string>
    </property>
    <property name="text">
     <string>Konwertuj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Konwersja sytemów liczbowych</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>dziesiętny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>binarny</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>szesnastkowy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>ósemkowy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>pamięć/wynik ---&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtoni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
